misc-tools/Makefile

```diff
@@ -25,6 +25,7 @@ $(SUBST_FILES): %: %.in $(TOPDIR)/paths.mk
 install-domserver:
 	$(INSTALL_PROG) -t $(DESTDIR)$(domserver_bindir) $(SUBST_DOMSERVER)
 	$(INSTALL_PROG) -t $(DESTDIR)$(domserver_bindir) import-contest
+	$(INSTALL_PROG) -t $(DESTDIR)$(domserver_bindir) dj_utils.py
```

Comment:
I think this should be installed in `libdir` instead, as this file will not be called directly.

Reply:
How would that import then work in python?

Reply:
Good question, I don't know. How does this work for other (standard) libraries?

Reply:
I guess with something like https://stackoverflow.com/questions/135035/python-library-path inject the `domserver_libdir` path into each python binary using this library.

Reply:
It may be that one can modify the `sys.path`, if that works it would require us to make the `import-contest` and the `configure-domjudge` scripts template files.

Reply:
See https://github.com/DOMjudge/domjudge/pull/1745